shared/protocol/src/test/java/io/pravega/shared/NameUtilsTest.java

```diff
@@ -227,4 +227,29 @@ public void testGetConnectionDetails() {
         Assert.assertEquals("localhost", NameUtils.getConnectionDetails("localhost :12345")[0].trim());
         Assert.assertEquals("12345", NameUtils.getConnectionDetails("localhost :12345")[1].trim());
     }
+
+    @Test
```

Comment:
Add timeout for test

Reply:
This uses a globalTimeout at the top of the file.